app/src/main/java/com/android/shelfLife/ui/camera/CameraView.kt

```diff
@@ -259,6 +353,123 @@ fun PermissionDeniedScreen(navigationActions: NavigationActions) {
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Top) {
+          Spacer(modifier = Modifier.height(16.dp))
+
+          ExposedDropdownMenuBox(
+              expanded = locationExpanded,
+              onExpandedChange = { locationExpanded = !locationExpanded }) {
+                OutlinedTextField(
+                    value = location.name.lowercase(),
+                    onValueChange = {},
+                    label = { Text("Location") },
+                    readOnly = true,
+                    trailingIcon = {
+                      ExposedDropdownMenuDefaults.TrailingIcon(expanded = locationExpanded)
+                    },
+                    modifier = Modifier.fillMaxWidth().menuAnchor())
+                ExposedDropdownMenu(
+                    expanded = locationExpanded, onDismissRequest = { locationExpanded = false }) {
+                      FoodStorageLocation.values().forEach { selectionOption ->
+                        DropdownMenuItem(
+                            text = { Text(selectionOption.name) },
+                            onClick = {
+                              location = selectionOption
+                              locationExpanded = false
+                            })
+                      }
+                    }
+              }
+
+          // For dates a future improvement could be having a calendar interface rather than manual
+          // input
+          OutlinedTextField(
+              value = expireDate,
+              onValueChange = { expireDate = it },
+              label = { Text("Expire Date") },
+              placeholder = { Text("dd/mm/yyyy") },
+              modifier = Modifier.fillMaxWidth().padding(bottom = 16.dp),
+          )
+
+          OutlinedTextField(
+              value = openDate,
+              onValueChange = { openDate = it },
+              label = { Text("Open Date") },
+              placeholder = { Text("dd/mm/yyyy") },
+              modifier = Modifier.fillMaxWidth().padding(bottom = 16.dp),
+          )
+
+          OutlinedTextField(
+              value = buyDate,
+              onValueChange = { buyDate = it },
+              label = { Text("Buy Date") },
+              placeholder = { Text("dd/mm/yyyy") },
+              modifier = Modifier.fillMaxWidth().padding(bottom = 32.dp),
+          )
+
+          Button(
+              onClick = {
+                val newFoodItem =
+                    FoodItem(
+                        uid = foodItemViewModel.getUID(),
+                        foodFacts = foodFacts,
+                        location = location,
+                        expiryDate = formatDateToTimestamp(expireDate),
+                        openDate = formatDateToTimestamp(openDate),
+                        buyDate = formatDateToTimestamp(buyDate),
+                        // Logic to determine status depending on the dates given
+                        status = FoodStatus.CLOSED)
```

Comment:
Not necessary to provide `status`
-> by default, `FoodItem `creates a item with a `FoodStatus.CLOSED` status
```suggestion
                       )
```